_data/releases.json

```
@@ -82,5 +44,43 @@
       "end": "Thu Feb 8 2024",
       "days": "2"
     }
+  },
+  "next_nightly": {
+    "version": "24.04",
+    "cudf_dev": {
+      "start": "1 19 2024",
```

I think the other dates use month abbreviations.
```suggestion
      "start": "Jan 19 2024",
```